commit 103ec440943e9108094e89510e04dcd299c0b21e
Author: Alistair Strachan <astrachan@google.com>
Date:   Wed Jan 23 12:06:06 2019 -0800

    arm64: dts: hikey: Revert "Enable HS200 mode on eMMC"
    
    commit 8d26c1390aec795d492b8de5e4437751e8805a1d upstream.
    
    This reverts commit abd7d0972a192ee653efc7b151a6af69db58f2bb. This
    change was already partially reverted by John Stultz in
    commit 9c6d26df1fae ("arm64: dts: hikey: Fix eMMC corruption regression").
    
    This change appears to cause controller resets and block read failures
    which prevents successful booting on some hikey boards.
    
    Cc: Ryan Grachek <ryan@edited.us>
    Cc: Wei Xu <xuwei5@hisilicon.com>
    Cc: Manivannan Sadhasivam <manivannan.sadhasivam@linaro.org>
    Cc: Rob Herring <robh+dt@kernel.org>
    Cc: Mark Rutland <mark.rutland@arm.com>
    Cc: linux-arm-kernel@lists.infradead.org
    Cc: devicetree@vger.kernel.org
    Cc: stable <stable@vger.kernel.org> #4.17+
    Signed-off-by: Alistair Strachan <astrachan@google.com>
    Signed-off-by: John Stultz <john.stultz@linaro.org>
    Signed-off-by: Wei Xu <xuwei5@hisilicon.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/arch/arm64/boot/dts/hisilicon/hi6220-hikey.dts b/arch/arm64/boot/dts/hisilicon/hi6220-hikey.dts
index e47609aa20d6..e80a792827ed 100644
--- a/arch/arm64/boot/dts/hisilicon/hi6220-hikey.dts
+++ b/arch/arm64/boot/dts/hisilicon/hi6220-hikey.dts
@@ -301,7 +301,6 @@
 
 		dwmmc_0: dwmmc0@f723d000 {
 			cap-mmc-highspeed;
-			mmc-hs200-1_8v;
 			non-removable;
 			bus-width = <0x8>;
 			vmmc-supply = <&ldo19>;
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1128_103ec440943e9108094e89510e04dcd299c0b21e.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1128_103ec440943e9108094e89510e04dcd299c0b21e.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
